app/metrics.yaml

```diff
@@ -130,3 +130,66 @@ settings_screen:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-08-17"
+
+shortcuts:
+  shortcuts_on_home_number:
+    type: quantity
+    description: |
+      The number of shortcuts the user has on home screen,
+      0, 1, 2, 3 or 4 (maximum)
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5056
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5189
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-20"
+    unit: shortcut(s)
+  shortcut_opened_counter:
+    type: counter
+    description: |
+      A counter that indicates how many times a user has opened
+      a website from a shortcut in the home screen.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5056
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5189
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-20"
+  shortcut_added_counter:
```

Comment:
Same recommendations as above about "per session" (send it on a custom ping), or "per day" (this is fine, then).

Reply:
Same as the others.